src/transformers/models/llava_next/image_processing_llava_next.py

```diff
@@ -516,6 +517,8 @@ def preprocess(
                 - `"channels_first"` or `ChannelDimension.FIRST`: image in (num_channels, height, width) format.
                 - `"channels_last"` or `ChannelDimension.LAST`: image in (height, width, num_channels) format.
                 - `"none"` or `ChannelDimension.NONE`: image in (height, width) format.
+            concat_images (`bool`, default to True):
```

Comment:
To not have surprising new behaviour for batched generation, I would remove this flag and instead pad the images up to the max number of patches in a batch, so that pixel_values will always be of shape `(batch_size, max_number_of_patches, 3, height, width)`